containerfiles/Containerfile.ci

```diff
@@ -5,3 +5,5 @@ USER root
 # workaround for RHBZ#2184640
 RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial
 RUN dnf update -y && dnf install -y git python3-pip python3-netaddr make gcc sudo rsync && yum clean all
+RUN mkdir -p /home/zuul/src/github.com/openstack-k8s-operators
+RUN git clone https://github.com/openstack-k8s-operators/install_yamls /home/zuul/src/github.com/openstack-k8s-operators/install_yamls
```

Comment:
can we export `/home/zuul/src/github.com/openstack-k8s-operators` as a var so we can reuse it inside the Containerfile?

Reply:
Any actual usecase ?

Reply:
vars makes sense when it is used >1 and I only see it once in this file

Reply:
in fact, we are using that dir twice in 2 lines. It's not so important BTW, it could be useful if we needed to change it but it's not the case for now I guess.